src/blink-api.js

```diff
@@ -574,8 +574,8 @@ class BlinkAPI {
      *   ]
      * }
      **/
-    async getMediaChange(maxTTL = 60, after = "1970-01-01T00:00:01+0000", page = 1) {
-        return await this.get(`/api/v1/accounts/{accountID}/media/changed?since=1970-01-01T00:00:01+0000&page=1`, maxTTL);
+    async getMediaChange(maxTTL = 60, after = "2021-02-01T00:00:01+0000", page = 1) {
```

Comment:
this is intentional default value

Reply:
I can revert back if necessary. It seemed to be only using this to get the media history to determine if there is an alert? If so does it need to go back to the full history?